src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  padding: 0 50px;
```

Comment:
According to the Code Style Rules, for padding and margin properties, if values are 0, the unit can be omitted. So it's better to write 'padding: 0;' when the value is zero.